chia/util/files.py

```diff
@@ -71,6 +73,8 @@ async def write_file_async(file_path: Path, data: Union[str, bytes], *, file_mod
     mode: Literal["w+", "w+b"] = "w+" if type(data) == str else "w+b"
     temp_file_path: Path
     async with tempfile.NamedTemporaryFile(dir=file_path.parent, mode=mode, delete=False) as f:
+        # Ignoring type error since it is not obvious how to tie the type of the data
+        # being passed in to the type of the file object, etc.
         temp_file_path = f.name  # type: ignore[assignment]
         await f.write(data)  # type: ignore[arg-type]
```

Comment:
don't you get a warning about "unused ignore" still?

Reply:
I assumed you wanted to make this change because of these:
```
chia/util/files.py:74: error: Unused "type: ignore" comment
chia/util/files.py:75: error: Unused "type: ignore" comment
Found 2 errors in 1 file (checked 748 source files)
```

Reply:
With mypy < 1 you will get such a message.  With mypy 1 you will not.  As far as I know.  One of those transitional things where you need to make sure your env is updated.

Reply:
@arvidn Unfortunately, we don't have the mypy version pinned, because they are still making fast progress, so you need the latest mypy in you venv.